### ISSUE TITLE
allow customizable texts in the progress bar

### ISSUE THREAD

**Issue by ekinakyurek** [cloud-oak/Tqdm.jl#3]
We may find a way to add customizable text, which changes in every iteration, to progress bar.
cc: @denizyuret

**Comment by cloud-oak** [comment]
You mean like tqdm's `set_description`?

**Comment by denizyuret** [comment]
We were wondering if some measure of neural net training progress (e.g. the
moving average of training loss) can be displayed somewhere on the same
line.

**Comment by cloud-oak** [comment]
Implemented by #5 , so closing
